express-api/src/controllers/common/IBaseEntity.ts

```diff
@@ -0,0 +1,8 @@
+import { UUID } from 'crypto';
+
+export interface IBaseEntity {
+  createdOn: string;
+  updatedOn: string;
+  createdById: UUID;
+  updatedById: UUID;
+}
```

Comment:
I think that the updated fields might be undefined at times if something had been created but never updated. They are also nullable in the database.
```suggestion
export interface IBaseEntity {
  createdOn: string;
  updatedOn?: string;
  createdById: UUID;
  updatedById?: UUID;
}
```